.github/workflows/code_checks.yml

```diff
@@ -0,0 +1,41 @@
+name: Core Tests
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
So for now this is fine, but I imagine that within the next few months (probably early September at the latest) that Ubuntu-latest will be Ubuntu 24. Might be worth making this ubuntu-22 (not a critical issue for sure).

Reply:
Addressed in recent commit